src/usr/lib/rsetup/cli/baota.sh

```diff
@@ -0,0 +1,11 @@
+# shellcheck shell=bash
+
+install_baota() {
+    __parameter_count_check 0 "$@"
+    curl -s https://download.bt.cn/install/install-ubuntu_6.0.sh | sudo bash /dev/stdin ed8484bec
```

Comment:
```suggestion
    sudo bash < <(curl -s "https://download.bt.cn/install/install-ubuntu_6.0.sh")
```
Please don't pin this to a specific commit. You know we ain't gonna come back, update, and validate for new commits, so just install the latest version, and redirect user's problem to Baota itself.